circle.yml

```diff
@@ -0,0 +1,47 @@
+machine:
+  php:
+    version: 5.5.21
+  ruby:
+    version: 2.1.3
+  environment:
+    APP_ENV: testing
+
+general:
+  artifacts:
+    - "tests/codeception/_output"
+    - "storage/logs"
+
+dependencies:
+  cache_directories:
+    - elasticsearch-1.6.0
+    - node_modules
+    - public/components
+    - vendor
+  pre:
+    # - sed -i 's/^;//' ~/.phpenv/versions/$(phpenv global)/etc/conf.d/xdebug.ini
+    - echo "memory_limit = 1024M" > ~/.phpenv/versions/$(phpenv global)/etc/conf.d/memory.ini
+    - if [[ ! -e elasticsearch-1.6.0 ]]; then wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.6.0.tar.gz && tar -xvf elasticsearch-1.6.0.tar.gz; fi
+    - elasticsearch-1.6.0/bin/elasticsearch: {background: true}
```

Comment:
The elasticsearch version is in here a few times. I would move this to a separate .sh file for easy changing. I believe this file already exists for the vagrant box: https://github.com/madewithlove/skeletor/blob/5.1/vagrant/scripts/provision/elasticsearch.sh

If we use the same script, its easier to manage (I do this for elasticsearcher).